app/controllers/ops_controller/settings/ldap.rb

```diff
@@ -328,6 +326,21 @@ def ls_delete
 
   private
 
+  def checked_or_params_id(klass)
+    objs = find_checked_items
+    obj = objs.blank? && params[:id] ? params[:id] : objs[0]
```

Comment:
could you use `params[:id].present?` here, so it's clear what are you testing it for?